package.json

```diff
@@ -32,13 +32,20 @@
     "format:check": "prettier --check \"**/*.+(json|yml|yaml|css|md)\"",
     "flow:check": "flow check",
     "concurrently": "concurrently",
-    "postinstall": "husky install"
+    "postinstall": "node -e \"process.env.NODE_ENV != 'production' && process.exit(1)\" || husky install"
```

Comment:
One option could be `prepublish` script but this seems to be ok too.

Reply:
changed :+1: